commit 3c7792eceb4a9e8507004a2ab1c6b2bb66d8d100
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Fri Mar 16 10:13:19 2018 +0000

    BCM27XX_DT: Delete 'stdout-path' property
    
    The 'stdout-path' property introduced upstream changes the behaviour
    on a non-Bluetooth Pi by enabling a console on UART0 when the user
    may not want one. The boot order is such that /dev/console ends up
    on the serial port instead of tty0.
    
    Delete the property in downstream DTBs to retrurn to the previous
    behaviour of only enabling consoles selected in cmdline.txt.
    
    See: https://github.com/raspberrypi/linux/issues/2436
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm270x.dtsi b/arch/arm/boot/dts/bcm270x.dtsi
index 5c12450d2c30..d26b10ad0245 100644
--- a/arch/arm/boot/dts/bcm270x.dtsi
+++ b/arch/arm/boot/dts/bcm270x.dtsi
@@ -4,6 +4,7 @@
 / {
 	chosen {
 		bootargs = "";
+		/delete-property/ stdout-path;
 	};
 
 	soc: soc {
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1207_3c7792eceb4a9e8507004a2ab1c6b2bb66d8d100.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1207_3c7792eceb4a9e8507004a2ab1c6b2bb66d8d100.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
